docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: guide
+summary_title: "Codewind in CodeReady Workspaces"
+title: "Getting Started with Codewind in CodeReady Workspaces"
+categories: guides 
+description: "Use CodeReady Workspaces to develop cloud-native applications from an OpenShift cluster."
+permalink: codewind-crw-quick-guide.html
+duration: 10 minutes
+keywords: Codewind, CodeReady Workspaces, OpenShift
+objectives: ["Install CodeReady Workspaces and Codewind.", "Create a Codewind workspace within CodeReady Workspaces."]
+icon: images/learn/icon_cloud.svg  
+---
+
+## Overview
+Use Eclipse Codewind to develop microservice applications from application stacks in an integrated developer environment (IDE). CodeReady Workspaces provides a containerized IDE for cloud-native application development on an OpenShift cluster. 
+
+## Developing with CodeReady Workspaces 
+CodeReady Workspaces uses Kubernetes and containers to provide a preconfigured environment. Use CodeReady Workspaces to create, build, and test your code in OpenShift containers but feel like you are working on an IDE on your local machine.  
+
+### Prerequisite
+CodeReady Workspaces require at least two 5Gi ReadWriteOnce (RWO) persistent volumes on the cluster to install and a 5Gi RWO volume for each created workspace.
```

Comment:
We should highlight that OpenShift itself is prerequisite for this lab. And maybe recommend "CodeReady Containers" (https://code-ready.github.io/crc/) if they don't have OpenShift already available.

Reply:
Thanks, John. I'll update the prereqs this afternoon. If all else looks good to you, I can see if Michael can merge tomorrow morning.

Reply:
Yup, everything else looks good